book/src/sprites.md

```diff
@@ -0,0 +1,16 @@
+# Sprites
+
+Sprites are 2D images that represent an object or background. Sprites are represented by two main chunks of data:
+
+* **Texture:** The image made of pixels.
+* **Layout:** The coordinates of the sprites on that image.
```

Comment:
Not a big fan of the use of the word layout, it doesn't convey much. There might be a better word for this?

Reply:
hm, my mind is blank on choosing another single word; "sprite positions" kind of conveys it, but it's 2 words. Shall ask on the chat and leave this comment open in case someone else has a better thesaurus

Reply:
That works for me, just find layout to be a bit generic and kind hard to attach. I'm fine with keeping it though if thats what the majority want.